eng/versioning.targets

```diff
@@ -59,21 +59,28 @@
 
   <ItemGroup>
     <_unsupportedOSPlatforms Include="$(UnsupportedOSPlatforms)" />
+    <_supportedOSPlatforms Include="$(SupportedOSPlatforms)" />
   </ItemGroup>
 
-  <!-- Adds UnsupportedOSPlatform attribute to the assembly when:
-      * At least one <UnsupportedOSPlatforms /> has been specified
+  <!-- Adds UnsupportedOSPlatform and SupportedOSPlatform attributes to the assembly when:
+      * At least one <UnsupportedOSPlatforms /> or <SupportedOSPlatforms /> has been specified
```

Comment:
Note: Once this PR goes through, I plan to send a follow-up PR that utilizes the `<SupportedOSPlatforms>` property instead of `<IsWindowsSpecific>`. https://github.com/jeffhandley/runtime/commit/0f530be2ae763cbf5d0a2e70fb62def351549110